test/pdfs/.gitignore

```diff
@@ -278,6 +278,7 @@
 !issue5070.pdf
 !issue5238.pdf
 !issue5244.pdf
+!hsbc.pdf
```

Comment:
This should *probably* be a linked test-case instead, looking at the file contents.